docs/edgeql/functions/converters.rst

```diff
@@ -49,3 +385,251 @@ the functions below allow more options for such conversions.
             "a": 2,
             "b": "hello"
         }'}
+
+
+Formatting
+----------
```

Comment:
This is a top level section, and "Date and time formatting options" is second level?  If so, this should have different underline style.

Reply:
I've changed the nesting level for the sections.